controllers/tests/controller/selfnoderemediation_controller_test.go

```diff
@@ -167,7 +167,7 @@ var _ = Describe("SNR Controller", func() {
 			It("Remediation flow", func() {
 				node := verifyNodeIsUnschedulable()
 
-				verifyEvent("Normal", "RemediationStarted", "Remediation started")
+				verifyEvent("Normal", "RemediationStarted", "Remediation started by SNR manager")
```

Comment:
How about using _EventTypeNormal_ and _EventTypeNormal_  from https://github.com/kubernetes/api/blob/v0.29.2/core/v1/types.go#L6354-L6359 instead of typing the string? Here and in other occurrences in this file.

Reply:
Instead of using the string _RemediationStarted_ we can also use the [common remediation event reasons](https://github.com/medik8s/common/blob/main/pkg/events/events.go#L15-L18).

Reply:
I don't think it's very significant in test code.
One can even argue against it in test code, because it would be safer to validate vs a "decoupled " hardcoded value (personally not an issue as well IMO).

Reply:
What about the event type?

Reply:
TBH my original response for the event type question  :sweat_smile: (I only saw the followup after I posted)
I was just luck that my answer fitted nicely the second question